src/worker/index.ts

```diff
@@ -48,6 +48,8 @@ self.addEventListener('activate', event => {
     event.waitUntil(self.clients.claim());
 });
 
+self.addEventListener('fetch', () => console.log('fetch'));
```

Comment:
remove

Reply:
Removing this will cause the service worker to no longer be valid for PWA installation and fail to run on iOS. The fetch event has to be present for the worker to be valid, but I'm not particularly aware of anything we're interested in doing with fetched data. For the most part this listener is for updating existing cached data, but we have other methods of achieving the same currently, looking at the existing code. 

This is here simply to allow us to work on iOS.

Reply:
Yeah, it is definitely require for the service worker to be considered installable, and that's required for it to work as a PWA, which, in turn, is required to enable W3C push notifications on iOS D: